js/src/Dapps/blacklist.js

```diff
@@ -0,0 +1,14 @@
+module.exports = [
+  // tokendeploy
+  '0xf9f2d620c2e08f83e45555247146c62185e4ab7cf82a4b9002a265a0d020348f',
```

Comment:
where are these hashes coming from?

Reply:
They are the actual ids are registered in the registries. (Initially created using the sha3 of the name, same across all chains)